scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html

```diff
@@ -465,3 +468,13 @@ <h5 class="modal-title" id="archiveCaseLabel">Confirm archive case</h5>
   </div>
 </div>
 {% endmacro %}
+
+{% macro coverage_qc_report(institute, case) %}
+<div href="#" class="bg-dark list-group-item text-white">
+  <div class="d-flex flex-row flex-fill bd-highlight"></span>
+    <span class="menu-collapsed ml-2">Coverage and QC report
+      <a href="{{ url_for('cases.coverage_qc_report', institute_id=institute,
+                        case_name=case.display_name, format='html') }}" target="_blank"><i class="fa fa-file-pdf-o"></i></a>
```

Comment:
Hm, so here you might want to do two things: 
- use fa-link for the html link
- in contrast to the CNV report, also add a pdf generation link like for the case delivery report